docroot/themes/custom/uids_base/scss/components/uiowa-bar.scss

```diff
@@ -1,10 +1,10 @@
-@import "assets/scss/_variables.scss";
-@import "assets/scss/_utilities.scss";
-//@import "components/03 - global/branding-bar/uiowa-apply/_uiowa-apply.scss";
-//@import "components/03 - global/branding-bar/uiowa-links/_uiowa-links.scss";
-//@import "components/03 - global/branding-bar/uiowa-search/_uiowa-search.scss";
-//@import "components/03 - global/branding-bar/uiowa-bar/_uiowa-bar.scss";
-//@import "components/03 - global/branding-bar/uiowa-search/_uiowa-search.scss";
+@import "uids3/assets/scss/_variables.scss";
+@import "uids3/assets/scss/_utilities.scss";
+//@import "uids3/components/03 - global/branding-bar/uiowa-apply/_uiowa-apply.scss";
+//@import "uids3/components/03 - global/branding-bar/uiowa-links/_uiowa-links.scss";
+//@import "uids3/components/03 - global/branding-bar/uiowa-search/_uiowa-search.scss";
+//@import "uids3/components/03 - global/branding-bar/uiowa-bar/_uiowa-bar.scss";
+//@import "uids3/components/03 - global/branding-bar/uiowa-search/_uiowa-search.scss";
```

Comment:
Seems like we can remove this since it is out of date with how we are naming things now.